driver-core/src/test/resources/client-side-encryption/README.rst

```diff
@@ -71,26 +71,28 @@ Each YAML file has the following keys:
 
   - ``clientOptions``: Optional, parameters to pass to MongoClient().
 
-    - ``auto_encrypt_opts``: Optional
+    - ``autoEncryptOpts``: Optional
```

Comment:
Having the diffs for the readme is super useful :)

Are there JSON test updates as well?

Reply:
No, looks like the tests did not change.  The README was just updated to reflect what's in the json files.